### ISSUE TITLE
linux docker image build failing

### ISSUE THREAD

**Issue by isuruf** [conda-forge/docker-images#88]
https://travis-ci.org/conda-forge/docker-images/jobs/511844961

cc @ocefpaf

**Comment by ocefpaf** [comment]
Looks like we are not pulling these from `defaults`:

```
PackagesNotFoundError: The following packages are not available from current channels:
  - conda-forge::binutils_impl_linux-64
  - conda-forge::libgcc-ng
  - conda-forge::gfortran_linux-64
  - conda-forge::binutils_linux-64
  - conda-forge::gfortran_impl_linux-64
  - conda-forge::libstdcxx-ng
```

Now that we have `strict` working nicely we should avoid as much as we can to vendorize those in `conda-forge`. I'll send a PR to fix this here. Thanks for the ping.

**Comment by isuruf** [comment]
Fixed in #90
